src/test/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/VisitInformationRepositoryTest.java

```diff
@@ -48,7 +48,7 @@ public void findAll() {
         assertThat(visits).extracting(VisitInformation::getLocation).containsExactly("Visiting Room", "Visiting Room", "Visiting Room", "Chapel", "Visiting Room", "Visiting Room", "Visiting Room", "Visiting Room", "Visiting Room", "Chapel", "Visiting Room", "Chapel", "Visiting Room", "Visiting Room", "Visiting Room");
         assertThat(visits).extracting(VisitInformation::getVisitType).containsExactly("SCON", "SCON", "SCON", "OFFI", "SCON", "SCON", "SCON", "SCON", "SCON", "OFFI", "SCON", "OFFI", "SCON", "SCON", "SCON");
         assertThat(visits).extracting(VisitInformation::getVisitTypeDescription).containsExactly("Social Contact", "Social Contact", "Social Contact", "Official Visit", "Social Contact", "Social Contact", "Social Contact", "Social Contact", "Social Contact", "Official Visit", "Social Contact", "Official Visit", "Social Contact", "Social Contact", "Social Contact");
-        assertThat(visits).extracting(VisitInformation::getLeadVisitor).containsExactly("JESSY SMITH1", null, null, null, null, null, null, null, null, null, null, null, null, null, "JESSY SMITH1");
+        assertThat(visits).extracting(VisitInformation::getLeadVisitor).containsExactly("JESSY SMITH1", " ", " ", " ", " ", " ", " ", " ", " ", " ", " ", " ", " ", " ", "JESSY SMITH1");
```

Comment:
This was previously nulls because `concat_nulls` was false on HSQLDB. 

When running the same SQL on NOMIS it returns empty spaces as per this test (because the SQL select is `P.FIRST_NAME || ' ' || P.LAST_NAME LEAD_VISITOR,`).